Mathlib/GroupTheory/GroupAction/SubMulAction.lean

```diff
@@ -329,6 +329,15 @@ theorem val_image_orbit {p : SubMulAction R M} (m : p) :
 lemma orbit_of_sub_mul {p : SubMulAction R M} (m : p) :
     (mul_action.orbit R m : set M) = MulAction.orbit R (m : M) := rfl
 -/
+
+theorem val_preimage_orbit {p : SubMulAction R M} (m : p) :
+    Subtype.val ⁻¹' MulAction.orbit R (m : M) = MulAction.orbit R m := by
+  rw [← val_image_orbit, Subtype.val_injective.preimage_image]
+
+lemma mem_orbit_subMul_iff {p : SubMulAction R M} {x m : p} :
+    x ∈ MulAction.orbit R m ↔ (x : M) ∈ MulAction.orbit R (m : M) := by
+  rw [←val_preimage_orbit, Set.mem_preimage]
```

Comment:
I added this first one, since it seems possibly a little more direct that the one expressed in terms of membership. Do you still want to keep both?

Reply:
I think both are useful, yes.